### PR TITLE
Revert "live: Call webbrowser.open"

### PR DESCRIPTION
**Pull request by efiop**
Reverts iterative/dvc#6913

See https://github.com/iterative/dvc/commit/318b269ad118f198bb97b203e151529398da0f54#r60727503 and https://github.com/iterative/dvc/pull/6913#discussion_r754003524

### CODE REVIEW

**Review by daavoo — COMMENTED** [review]
https://github.com/iterative/dvc/pull/7032 Should fix the issue

File: dvc/repo/live.py [hunk 1]
```diff
@@ -14,27 +9,15 @@
     from dvc.output import Output
     from dvc.repo import Repo
 
-webbrowser_open = once_per_args(webbrowser.open)
```
Comment (skshetry) [inline dvc/repo/live.py:17]: Might be easy to just use `autouse=` mocked object than the rollback. Let me see.

**Comment by efiop** [comment]
Closing in favor of #7032

File: dvc/repo/live.py (hunk 1 above)
Comment (efiop) [reply dvc/repo/live.py:17]: @skshetry Let's continue in https://github.com/iterative/dvc/pull/7032

**Pull request closed without merging**